.gitignore

```diff
@@ -12,3 +12,146 @@ cmake_uninstall.cmake
 *.pdf
 *.swp
 tests/TestFiles*.tar.gz
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
+
+# Distribution / packaging
+.Python
+build/
```

Comment:
build/ already exist in `.gitignore`

Reply:
I just append the standard python gitignore